.github/workflows/e2e.yml

```diff
@@ -41,6 +41,13 @@ jobs:
           DEBUG: pw:webserver
           HAPPO_API_KEY: ${{ secrets.HAPPO_API_KEY }}
           HAPPO_API_SECRET: ${{ secrets.HAPPO_API_SECRET }}
+          # When a pull request is made from a fork, the secrets above are 
+          # not available for security reasons, so happo cannot use them. This 
+          # value enables happo to use a fallback behavior where it generates
+          # a one-off token for this use case.
+          # https://github.com/happo/happo-e2e/blob/master/src/loadHappoConfig.js
+          # https://github.com/happo/happo.io/blob/master/src/loadUserConfig.js
+          CHANGE_URL: ${{  github.event_name == 'pull_request' && github.event.pull_request.html_url }}
```

Comment:
We should only need this for PRs. Merges to `main` should always have access to the secrets.